Les cycles du développement piloté par les tests (TDD)

Moosh-be opened the pull request.

Moosh-be committed:
Les cycles du développement piloté par les tests (TDD)

nmereaux commented on _posts/2015-01-06-les-cycles-du-developpement-pilote-par-les-tests.markdown:
```diff
@@ -45,17 +45,17 @@ Une autre manière de considérer cette idée, c'est :
 > Au fur et à mesure que les tests deviennent plus spécifiques, le code devient plus générique.  
 
-Au fur et à mesure qu'une suite de tests grossie, elle devient plus spécifique, c'est à dire qu'elle devient une _spécification_ du comportement encore plus détaillée. Les bons développeurs d'applications font face à cet accroissement de spécifications par un accroissement de la _généricité_ de leur code. Pour le dire autrement : _les développeurs font fonctionner des cas spécifiques en écrivant  ...
+Au fur et à mesure qu'une suite de tests grossie, elle devient plus spécifique, c'est à dire qu'elle devient une _spécification_ du comportement encore plus détaillé. Les bons développeurs d'applications font face à cet accroissement de spécifications par un accroissement de la _généricité_ de leur code. Pour le dire autrement : _les développeurs font fonctionner des cas spécifiques en écrivant d ...
```
C'est la spécification qui est détaillée, la terminaison en "ée" est correcte

Moosh-be committed:
la terminaison en "ée" est correcte

nmereaux Nov 10, 2020 Contributor
C'est la spécification qui est détaillée, la terminaison en "ée" est correcte
